src/pages/components/tile/style.mdx

```diff
@@ -33,11 +118,21 @@ import { white } from '@carbon/themes';
 |         | min-width         | 128 / 8  | –             |
 |         | padding (minimum) | 16 / 1   | `$spacing-05` |
 
-<div className="image--fixed">
+<Row>
+<Column colLg={12}>
+
+![Structure and measurements for tiles](images/tile-style-11.png)
```

Comment:
This image has a typo. Should say `modes`
<img width="770" alt="Screen Shot 2023-03-01 at 1 24 25 PM" src="https://user-images.githubusercontent.com/43969356/222229474-279c76cd-d0bc-4f4a-bb4e-ffd0fe406d10.png">